consts/index.js

```diff
@@ -1 +1,10 @@
 export const HEADER = 'header'
+
+export const META_BASE_TITLE = 'Data Version Control · DVC'
```

Comment:
This seems to be causing build error https://reactjs.org/docs/error-decoder.html/?invariant=31&args[]=object%20with%20keys%20%7BMETA_BASE_TITLE%7D&args[]=.

See https://dashboard.heroku.com/apps/dvc-org-pr-801/activity/builds/c32b9281-33f1-4ae5-b4a3-2168468b6633. Probably easier to debug locally though (with the React Developer Tools browser extension).

Reply:
Also, I wonder whether we should just put this file in the root dir or in src/ as consts.js

Reply:
It was caused by a wrong JSX expression, addressed in https://github.com/iterative/dvc.org/pull/801/commits/18b22324e2a02a5fe774f0fb5b33a1d4493b62e2

> Also, I wonder whether we should just put this file in the root dir or in src/ as consts.js

👍 done in https://github.com/iterative/dvc.org/pull/801/commits/690ec582e32a8f85e2e29528c63a9f3111bfd13d